language/move-prover/src/boogie_helpers.rs

```diff
@@ -181,6 +183,8 @@ fn boogie_well_formed_expr_impl(
             PrimitiveType::Num => conds.push(format!("$IsValidNum({})", name)),
             PrimitiveType::Bool => conds.push(format!("is#Boolean({})", name)),
             PrimitiveType::Address => conds.push(format!("is#Address({})", name)),
+            // TODO fix this for a real boogie check
+            PrimitiveType::Signer => conds.push(format!("is#Address({})", name)),
```

Comment:
A `panic!` here might be easier for Prover devs (cc @wrwg)?

Reply:
Yes, I was unsure what to do here for the prover, let me know and I'll do whatever y'all suggest
@junkil-park @wrwg @DavidLDill @shazqadeer

Reply:
Because compilation of all stdlib sources to boogie is part of our tests (and the MIRAI bot), a panic might be actually not the best choice.